### DISCUSSION TITLE
Question on footnote syntax and conversion to DOCX

### DISCUSSION THREAD

**Discussion by sbamberger** [jgm/pandoc#9715] [Q&A] [answered]
My preferred text editor IA Writer uses a Markdown syntax of [^Footnote text.], with the caret inside the square brackets. The footnote is not converting correctly in Pandoc when converting to a DOCX file — Word just sees it as text and not a footnote.

If I change the syntax to ^[Footnote text], with the caret preceding the square brackets, then Pandoc seems to convert it correctly to DOCX (Word sees it as a footnote). However, that syntax doesn't work well in my editor.

Is there any way to get Pandoc to respect the syntax with the caret inside the square brackets and convert it correctly to DOCX? Or am I stuck altering the source file in order to convert it?

**Comment by denismaier** [comment]
You could use a custom reader or maybe a filter.

Gesendet von Outlook für Android<https://aka.ms/AAb9ysg>
________________________________
From: sbamberger ***@***.***>
Sent: Thursday, May 2, 2024 8:08:16 PM
To: jgm/pandoc ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [jgm/pandoc] Question on footnote syntax and conversion to DOCX (Discussion #9715)


My preferred text editor IA Writer uses a Markdown syntax of [^Footnote text.], with the caret inside the square brackets. The footnote is not converting correctly in Pandoc when converting to a DOCX file — Word just sees it as text and not a footnote.

If I change the syntax to ^[Footnote text], with the caret preceding the square brackets, then Pandoc seems to convert it correctly to DOCX (Word sees it as a footnote). However, that syntax doesn't work well in my editor.

Is there any way to get Pandoc to respect the syntax with the caret inside the square brackets and convert it correctly to DOCX? Or am I stuck altering the source file in order to convert it?

**Comment by jgm** [comment]
Note that you can also create a footnote this way (non-inline):

```
this is my note.[^foo]

[^foo]:  Text of note here.
```

If IAWriter is really using `[^footnote text]` as an *inline* footnote syntax, bad on them for reinventing the wheel and contributing to incompatible syntaxes.

**Comment by sbamberger** [comment, marked as answer]
Thanks for the pointers. I think this got me to the correct answer.

IA Writer seems to support _only_ the regular footnote and not the inline style. I was using the regular footnote style and expecting it to render like an inline reference.

The work-around is pretty clear: Either use regular footnotes correctly in IA Writer, or alternatively use the inline style and just don't expect IA Writer to render inline correctly.

I've passed this on as a feature request to IA writer.

Thank you!
